doc/src/quick_start_guide.md

```diff
@@ -11,11 +11,11 @@ the following lines to your `Cargo.toml`:
 ```toml
 # The generated code depends on lalrpop-util.
 [dependencies]
-lalrpop-util = "0.21.0"
+lalrpop-util = "0.22.0"
 
 # Add a build-time dependency on the lalrpop library:
 [build-dependencies]
-lalrpop = "0.21.0"
+lalrpop = "0.22.0"
 # If you are supplying your own external lexer you can disable default features so that the
 # built-in lexer feature is not included
 # lalrpop = { version = "0.21.0", default-features = false }
```

Comment:
Maybe an easy change in the perl script but this comment always seems to be left out and its version needs to be updated manually.

Reply:
The version.sh script is a bit of a pain.  It has weird special handling for all the different variations on how the version line can be written, and tries to enumerate which files use this syntax.  That syntax is supported in all Cargo.toml files in docs, but not in the doc md files.  I'll fix the one liner here and rather than adding another exception case to the version.sh script, I'll see about just writing a single regex that handles all the formats and feed all the files to it.  There's not so much variation that that doesn't sound feasible.

Reply:
See #958.  If someone else wants to take a stab at a better approach, feel free.